web/modules/custom/rdf_entity/src/Entity/RdfEntitySparqlStorage.php

```diff
@@ -734,8 +735,9 @@ protected function doSave($id, EntityInterface $entity) {
       $id = $this->generateId();
       $entity->{$this->idKey} = (string) $id;
     }
-    elseif ($entity->isNew() && $this->idExists($id)) {
-      throw new \InvalidArgumentException("Attempting to create a new entity with the ID '$id' already taken.");
+    // The ID cannot overlap an existing 'rdf_entity' or 'taxonomy_term'.
```

Comment:
@sandervd, as I told you...

{code:php}
protected function idExists($id) {
  $query = <<<QUERY
ASK {
  <$id> ?field ?value
}
QUERY;
  return $this->sparql->query($query)->isTrue();
}
{code}

is lying. It returns TRUE in some cases where there's no entity with that ID. I cannot find the reason. Let's open a follow-up and fix the problem there. In the meantime I would not block migration on this.